test/Ocean/refvals/test_ocean_gyre_refvals.jl

```diff
@@ -1,12 +1,12 @@
 parr = [
     ["Q", "u[1]", 12, 12, 12, 12],
     ["Q", "u[2]", 12, 12, 12, 12],
-    ["Q", :η, 12, 12, 11, 12],
-    ["Q", :θ, 12, 12, 12, 12],
-    ["s_aux", :y, 12, 12, 12, 12],
-    ["s_aux", :w, 12, 12, 12, 12],
-    ["s_aux", :pkin, 12, 12, 12, 12],
-    ["s_aux", :wz0, 12, 11, 10, 12],
+    ["Q", "η", 12, 12, 11, 12],
+    ["Q", "θ", 12, 12, 12, 12],
+    ["s_aux", "y", 12, 12, 12, 12],
+    ["s_aux", "w", 12, 12, 11, 12],
```

Comment:
@blallen I'm not sure why this changed. I'd like to get to the bottom of it, but I'm not sure what's the best way. Any ideas?

Reply:
Also, is there a reason that sometimes symbols are used and other times strings?

Reply:
Ohhhh, I think I'm seeing the pattern, now..